cds_ils/config.py

```diff
@@ -411,6 +414,17 @@ def _parse_env_bool(var_name, default=None):
     "application/json": "cds_ils.series.serializers:json_v1_search",
     "text/csv": "cds_ils.series.serializers:csv_v1_search",
 }
+ILS_CIRCULATION_RECORDS_REST_ENDPOINTS[CIRCULATION_LOAN_PID_TYPE][
+    "record_serializers"
+] = {
+    "application/json": "cds_ils.circulation.serializers:json_v1_response",
+}
+ILS_CIRCULATION_RECORDS_REST_ENDPOINTS[CIRCULATION_LOAN_PID_TYPE][
+    "search_serializers"
+] = {
+    "application/json": "cds_ils.circulation.serializers:json_v1_search",
```

Comment:
Why not simply:
`ILS_CIRCULATION_RECORDS_REST_ENDPOINTS[CIRCULATION_LOAN_PID_TYPE]["search_serializers"]["text/csv"] = "cds_ils.circulation.serializers:csv_v1_search"`
So you don't need to touch the others.